test/unit/deprecated/lifecycle.js

```diff
@@ -1,113 +0,0 @@
-/* eslint-env jasmine, mocha */
```

Comment:
This used `ready()` and was testing stuff that the custom element polyfill should be tested to perform.